### DISCUSSION TITLE
How to know which result output corresponds to which bus or line?

### DISCUSSION THREAD

**Discussion by amaity** [e2nIEE/pandapower#1429] [Q&A] [answered]
Can anybody help out with a few basic things like - the results do not indicate either the names or the bus numbers given in the inputs. 

How do I know which output result corresponds to which input bus or line?? For example, can I simply take the name-column of the net.bus dataframe and assign it to net.res_bus dataframe? Can I do the same with the net.res_line df??

Thanks in advance.

**Comment by jkisse** [comment, marked as answer]
Yes, the `net.bus` and `net.res_bus` dataframe have exactly the same indices. So you could to something like `pd.concat([net.res_bus, net.bus.name], axis=1)` to have results and names in the same dataframe. Similar with all other result tables and their corresponding element tables.
